packages/salesforcedx-vscode-core/test/vscode-integration/util/orgUtil.test.ts

```diff
@@ -27,115 +27,131 @@ describe('orgUtil tests', () => {
   });
 
   describe('checkForExpiredOrgs tests', () => {
-    let today: Date;
+    const orgName1 = 'dreamhouse-org';
+    const orgName2 = 'ebikes-lwc';
+    const now = new Date();
+    const threeDaysFromNow = new Date(now.getTime() + 24 * 3 * 60 * 60000);
+    const yesterday = new Date(now.getTime() - 24 * 60 * 60000);
```

Comment:
There were a couple of problems in this test, but one was how we were building date strings.  For the last 3 days of the month it would end up being '22/08/32'ish.  I moved to calculating the value.

Reply:
These look nice.  Kinda wondering if they could be imported from some test util somewhere to be re-used in other tests needing dates...

Reply:
yeah good thinking.  Def seems like we could at the very least have a `DateService` that used moment or something for doing all the date things regardless b/c dates are jerks.